src/components/slider/styles/SSlider.tsx

```diff
@@ -0,0 +1,52 @@
+import styled, {css} from 'styled-components';
+import {MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND} from '../../../constants/styles';
+
+const Bronze = css<{ value: number; max: number; min: number }>`
+  
+  border-radius: 61px;
+  background: var(--ui-03);
+  width: 106px;
+  height: 4px;
+  position: relative;
+  
+  :before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    pointer-events: none;
+    width: ${({value, max, min}) => `${((value - min) / (max - min)) * 100}%`};
```

Comment:
@mahmoudtahadev  this is bad for css performance, please use a approach with transform3d instead